documentation/secure_storage.md

```diff
@@ -226,5 +231,8 @@ or crypto unit according to the method defined by your SoC vendor.
 
 ## Reference
 
-* [Secure Storage Presentation](http://www.slideshare.net/linaroorg/sfo15503-secure-storage-in-optee)
+* [SFO15 Secure Storage (slides)](http://connect.linaro.org/resource/sfo15/sfo15-503-secure-storage-in-op-tee/)
+* [LAS16 Secure Storage (slides)](http://connect.linaro.org/resource/las16/las16-504/)
+* [SFO17 Secure Storage (slides)](http://connect.linaro.org/resource/sfo17/sfo17-309/)
+* [HKG18 Secure Key Services (slides)](http://connect.linaro.org/resource/hkg18/hkg18-402/)
```

Comment:
I think this reference can be misleading to secure storage information. The "Secure Key Services" is not a secure storage implementation, rather a generic service that is based (for some parts) on OP-TEE secure storage solutions.

Reply:
@etienne-lms I agree. I overlooked the actual content and just checked the URL ;O